concent_api/core/payments/backends/sci_backend.py

```diff
@@ -55,6 +57,46 @@ def get_list_of_payments(
     return payments_list
 
 
+def are_all_payment_ts_younger_than_last_payment_closure_time_if_payment_exists(
+    requestor_eth_address: str,
+    provider_eth_address: str,
+    subtask_results_accepted_list: List[ForcedPaymentEvent],
+) -> bool:
+
+    oldest_payments_ts = _get_oldest_payment_timestamp_from_subtask_results_accepted_list(subtask_results_accepted_list)
+
+    forced_payment_event_list = _get_list_of_forced_payment_events(
+        requestor_eth_address=requestor_eth_address,
+        provider_eth_address=provider_eth_address,
+        search_payments_since_ts=oldest_payments_ts,
+    )
+
+    for forced_payment_event in forced_payment_event_list:
+        if oldest_payments_ts < forced_payment_event.closure_time:
+            return False
+    return True
+
+
+def _get_oldest_payment_timestamp_from_subtask_results_accepted_list(subtask_results_accepted_list: list) -> int:
+    return min(subtask_results_accepted.payment_ts for subtask_results_accepted in subtask_results_accepted_list)
```

Comment:
There is no need to create as simple one liner as a separate function if it is used only once.

Reply:
@rwrzesien, have you watched Uncle Bob's presentation? ;) 

It is ok (or even: is is _desired_), to extract some logical functionality to a separate function, regardless of how many times it is called